sda-commons-starter-web/src/test/java/org/sdase/commons/spring/boot/web/security/handler/MethodArgumentExceptionHandlerTest.java

```diff
@@ -27,14 +27,14 @@ class MethodArgumentExceptionHandlerTest {
   @ParameterizedTest
   @MethodSource("testData")
   void shouldConvertCustomDataToSnakeCase(String given, String expected) {
-    String actual = MethodArgumentExceptionHandler.camelToUpperSnakeCase(given);
+    String actual = ValidationExceptionHandler.camelToUpperSnakeCase(given);
     assertThat(actual).isEqualTo(expected);
   }
```

Comment:
Could you rename the test accordingly?

Reply:
renamed to `ValidationExceptionHandlerResponseTest.java`